_data/mentors.yml

```diff
@@ -231,7 +231,7 @@
 - name: Sonali Goel
   disabled: false
   matched: false
-  sort: 200
+  sort: 300
```

Comment:
I don't think Sonali will be more available this month. So please check with her first.

Reply:
Oh okay, it's because she recently filled the form to update her availability for July to November

Reply:
yes, unfortunately some personal issue came this week.

Reply:
☹️ I've removed her availability for July